herbert_core/classes/MPIFramework/messages/@aMessage/private/parse_payload_.m

```diff
@@ -0,0 +1,16 @@
+function payload_p = parse_payload_(payload)
+% Analyse the data structure, provided as input and modify the classes
+% which are not serializable into the form, accepting serialization
+%
+%
+% Itput:
+% ------
+%   payload -- the class, structure or variable to analyze
+%
+% Output:
+% -------
+%   payload_p  -- modified input, containing all serializable classes
+
+    payload_p = payload;
```

Comment:
doesn't appear to be doing any analysing at the moment. Maybe note to be done later or delegated to subclasses or whatever works here